ydb/docs/en/core/dev/troubleshooting/performance/schemas/_includes/overloaded-shards-diagnostics.md

```diff
@@ -0,0 +1,47 @@
+1. Analyze the **Overloaded shard count** chart in the **DB overview** Grafana dashboard.
+
+    ![](../_assets/overloaded-shards-dashboard.png)
+
+    The chart indicates whether the {{ ydb-short-name }} cluster has overloaded shards, but it does not specify which table shards are overloaded.
+
+1. To identify the table with an overloaded shard, follow these steps:
+
+    1. In the [Embedded UI](../../../../../reference/embedded-ui/index.md), go to the **Databases** tab and click on the database.
+
+    1. On the **Navigation** tab, ensure the required database is selected.
+
+    1. Open the **Diagnostics** tab.
+
+    1. Open the **Top shards** tab.
+
+    1. In the **Immediate** and **Historical** tabs, sort the shards by the **CPUCores** column and analyze the information.
+
+    ![](../_assets/partitions-by-cpu.png)
+
+    {% note info %}
+
+    You can also use system tables to get information about overloaded shards. For more information, see [History of overloaded partitions](../../../../system-views.md#top-overload-partitions).
+
+    {% endnote %}
+
+
+
+1. To pinpoint the schema issue, follow these steps:
+
+    1. Retrieve information about the problematic table using the [{{ ydb-short-name }} CLI](../../../../../reference/ydb-cli/index.md). Run the following command:
+
+        ```bash
+        ydb scheme describe <table_name>
+        ```
+
+    2. In the command output, analyze the **Auto partitioning settings**:
+
+        * `Partitioning by size`
+        * `Partitioning by load`
+        * `Max partitions count`
```

Comment:
This looks applicable only for row-oriented tables. Column-oriented ones would need a separate explanation.